generate_cert.py

```diff
@@ -64,6 +65,7 @@
     datetime.datetime.utcnow() + datetime.timedelta(days=2 * 365)
 ).add_extension(
     x509.SubjectAlternativeName([
+        x509.DNSName(u"android.magica-us.com"),
```

Comment:
Is this actually needed? We have wildcard `*.magica-us.com` below.

Reply:
oops... accidentally left in some debug code. just pushed another commit to yeet it out of existence.